apps/studio/pages/project/[ref]/settings/compute-and-disk.tsx

```diff
@@ -0,0 +1,28 @@
+import { DiskManagementForm } from 'components/interfaces/DiskManagement/DiskManagementForm'
+import SettingsLayout from 'components/layouts/ProjectSettingsLayout/SettingsLayout'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import type { NextPageWithLayout } from 'types'
+
+const AuthSettings: NextPageWithLayout = () => {
+  return (
+    <>
+      <ScaffoldContainer>
+        <ScaffoldHeader>
+          <ScaffoldTitle>Compute and Disk</ScaffoldTitle>
```

Comment:
do we need to explain why the disabled ones are disabled, in a tooltip maybe? 

![screenshot-2024-10-30-at-14 56 07](https://github.com/user-attachments/assets/f0751362-5125-4cf3-aa98-38f70023eeb9)

Reply:
agree. do it in a follow up though.